book/website/collaboration/research-infrastructure-roles/data-wrangler.md

```diff
@@ -0,0 +1,85 @@
+(cl-infrastructure-data-wranglers)=
+# Data Wranglers: Overview
+
+Data Wranglers can be viewed as a specialised type of Data Scientist, primarily working in the space between data generators and data analysts.
+The terminology used to describe various data science activities (for example data collection, wrangling, analysis, modelling, visualisation), and how they map onto different job titles, is domain specific and will vary on a project and organisational level.
+
+(cl-infrastructure-data-wranglers-tasks)=
+## What do Data Wranglers do?
+
+In a data science project, it is commonly observed that data wrangling tasks take the majority of time {cite:ps}`anaconda2020sods`, in contrast to data analysis and modelling.
+Traditionally, data wrangling tasks involve cleaning, restructuring and filtering data into standardised research ready formats.
+However, in terms of Data Wrangler as a profession, day to day tasks and objectives can be much more diverse.   
+
+Data Wranglers work primarily in the space between data generators and data analysts, who are addressing the research question of interest.
+Understanding the intended use for the data in the context of the analysis and research questions, Data Wranglers can be in the position of influencing data generators in improving data collection methods, ultimately benefiting all points of the data life cycle.
```

Comment:
```suggestion
Understanding the intended use for the data in the context of the analysis and research questions, Data Wranglers can be in the position of influencing data generators in improving data collection methods. 
This ultimately benefits all points of the data life cycle.
```
How does it benefit the data life cycle? Will the data be more reusable? Will the workflow be more efficient?

Reply:
I am actually going to delete the the _",ultimately benefiting all points of the data life cycle"_ phase as I think it  stops the flow of this paragraph and the point should be evident from what else we say in this paragraph 🤞

Reply:
See this commit [here](https://github.com/alan-turing-institute/the-turing-way/pull/2883/commits/ada9c65901259f90300073bc2add049ceff695f0)